libs/dmlf/tests/execution/local_executor_tests.cpp

```diff
@@ -131,6 +131,8 @@ TEST_F(LocalExecutorTests, delete_state)
 
 TEST_F(LocalExecutorTests, create_executable)
 {
+  // TODO(tfr): It is unclear why running the same code twice would yeild two different results
+  // Some explanation would be good.
   result_promise = CreateExecutable("HelloWorld", SourceFile{"hello_world.etch", hello_world_etch});
```

Comment:
executable names are unique (similarly to state names)

Reply:
Please add comments in the code to explain this as it is non-obvious when reading the test.